UI/menu/expanding.css

```diff
@@ -8,3 +8,42 @@ li.menu_closed > ul {
 li.menu_open > ul {
 	display: block;
 }
+
```

Comment:
Since all CSS is integrated into `UI/css/ledgersmb.css` and `UI/css/system/global.css`, I don't think that changing this file matters; more specifically, it probably needs to be removed?